### PR TITLE
edit.js - remove ability to clone non-owned avatar-entities

### PR DESCRIPTION
**Pull request by thoys**
Avatar entities that belong to another user should not be clonable with the Create app. This PR should fix that.

Internal bug tracker: https://highfidelity.fogbugz.com/f/cases/14068/Uncertified-avatar-entities-imported-by-one-user-can-be-copied-by-other-users

## QA test

Two users are required for the test. The domain should allow the users to rez.

1) User A: Use the `Edit -> Import Entities from URL` menu option and import `https://gist.githubusercontent.com/thoys/c4468e65affc5c3bc0fca5bd8c20b6e3/raw/6434c906c91d72170354549d0d668e9d35cdc868/AvatarEntity.json`
2) User B: Open Create app and select the imported entity
3) User B: Using hotkey (PC: Alt+ Left mouse click/Mac: Option+Left mouse click) drag select imported entity
4) The entity should not be cloned now. Please verify in the Create apps entities list that there is only one `avatar entity sphere`
5) User A: Open the create menu and create a cube.
6) User B: Using hotkey (PC: Alt+ Left mouse click/Mac: Option+Left mouse click) drag select imported entity
7) The entity should be cloned now (non-avatar-entity cloning should still function after this)

### CODE REVIEW

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -155,6 +154,20 @@ SelectionManager = (function() {
         that._update(true);
     };
 
+    that.duplicateSelection = function () {
+        var duplicatedEntityIDs = [];
+        Object.keys(that.savedProperties).forEach(function (otherEntityID) {
```
Comment (elisa-lj11) [inline scripts/system/libraries/entitySelectionTool.js]: no space between "function" and "("

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8858/app_12920_PR_b741888d7745872128b7ac830ad79349e5d76791.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8858/HighFidelity-Beta-PR12920-b741888d7745872128b7ac830ad79349e5d76791.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8858/HighFidelity-Beta-Interface-PR12920-b741888d7745872128b7ac830ad79349e5d76791.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8858/HighFidelity-Beta-PR12920-b741888d7745872128b7ac830ad79349e5d76791.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8858/HighFidelity-Beta-Interface-PR12920-b741888d7745872128b7ac830ad79349e5d76791.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by conklin94122** [comment]
@misslivirose Are you ok with this being merged?

**Comment by misslivirose** [comment]
@conklin94122 yep, please do

**Comment by conklin94122** [comment]
Done

**Pull request merged**